vaxrank/vaccine_peptide.py

```diff
@@ -189,8 +189,8 @@ def combined_score(self):
         return self.expression_score * self.mutant_epitope_score
 
     def to_dict(self):
+        epitope_predictions = self.mutant_epitope_predictions + self.wildtype_epitope_predictions
         return {
             "mutant_protein_fragment": self.mutant_protein_fragment,
-            "mutant_epitope_predictions": self.mutant_epitope_predictions,
-            "wildtype_epitope_predictions": self.wildtype_epitope_predictions,
+            "epitope_predictions": epitope_predictions,
```

Comment:
Shouldn't you also serialize/deserialize `num_mutant_epitopes_to_keep`? 


...also, why does that exist?

Reply:
it's for one of the outputs, the neoepitope Excel file - I think we maybe used this for Josh Brody's data before?